templates/vue/src/components/tyde/activities/CircleOfSupport/CircleView/index.vue

```diff
@@ -73,6 +73,28 @@
         src="https://twemoji.maxcdn.com/v/13.1.0/72x72/1f60a.png"
       />
     </div>
+    <div>
+      <avataaars
+        v-if="hasAvatar"
+        class="user"
+        :isCircle="avatar.isCircle"
+        :circleColor="avatar.circleColor"
+        :accessoriesType="avatar.accessoriesType"
+        :clotheType="avatar.clotheType"
+        :clotheColor="avatar.clotheColor"
+        :eyebrowType="avatar.eyebrowType"
+        :eyeType="avatar.eyeType"
+        :facialHairColor="avatar.facialHairColor"
+        :facialHairType="avatar.facialHairType"
+        :graphicType="avatar.graphicType"
+        :hairColor="avatar.hairColor"
+        :mouthType="avatar.mouthType"
+        :skinColor="avatar.skinColor"
+        :topType="avatar.topType"
+        :topColor="avatar.topColor"
+      ></avataaars>
+      <div v-else class="user">😊</div>
```

Comment:
Can we use a default avatar instead of an emoji to keep consistent with the styling?

Reply:
Replaced with default avatar